sanitize.py

```diff
@@ -0,0 +1,80 @@
+import re 
+from extract.utils import jid_to_abbr
+
+
+class Sanitizer:
+    """Abstract base class for sanitizing extracted text with pre-compiled regular expressions."""
+    def __init__(self):
+        self.re = None
+    def sanitize(self, text):
+        return self.re.sub('', text)
+
+class LineNumCleaner(Sanitizer):
+    """Removes line numbers from extracted text."""
+    def __init__(self):
+        self.re = re.compile(r"^ *line *\d+", flags=re.MULTILINE) 
+
+class NewlineCleaner(Sanitizer):
+    """Removes excessive newlines from extracted text."""
+    def __init__(self):
+        self.re = re.compile(r"\n(?=\n)")
+
+class TX_CSS_Cleaner(Sanitizer):
```

Comment:
super trivial, but snake case and all caps are usually for constants, maybe something like `TexasCSSCleaner` is better here